alibuild_helpers/cmd.py

```diff
@@ -10,10 +10,9 @@
 
 def execute(command, printer=debug):
   popen = subprocess.Popen(command, shell=is_string(command), stdout=subprocess.PIPE)
-  lines_iterator = iter(popen.stdout.readline, "")
-  for line in lines_iterator:
+  for line in iter(popen.stdout.readline, ""):
```

Comment:
Also this is unrelated to the PR. Can you please split it off?

Reply:
Undone.